osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -680,19 +927,23 @@ def start_benchmark(self):
 
         worker_assignments = calculate_worker_assignments(self.load_worker_coordinator_hosts, allocator.clients)
         worker_id = 0
+        # redline testing: keep track of the total number of workers
+        # and report this to the feedbackActor before starting a redline test
+        self.total_workers = 0
         for assignment in worker_assignments:
             host = assignment["host"]
             for clients in assignment["workers"]:
                 # don't assign workers without any clients
                 if len(clients) > 0:
+                    self.total_workers += 1
                     self.logger.info("Allocating worker [%d] on [%s] with [%d] clients.", worker_id, host, len(clients))
                     worker = self.target.create_client(host)
 
                     client_allocations = ClientAllocations()
                     for client_id in clients:
                         client_allocations.add(client_id, self.allocations[client_id])
                         self.clients_per_worker[client_id] = worker_id
-                    self.target.start_worker(worker, worker_id, self.config, self.workload, client_allocations)
+                    self.target.start_worker(worker, worker_id, self.config, self.workload, client_allocations, self.error_queue, self.queue_lock)
```

Comment:
I personally don't want to modify any logic with respect to existing functionality of OSB. It would be better to add a check here, if load testing enabled then send it with additional parameters else keep the original.